CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
 # Changelog for proto-route
-
-## Unreleased changes
+==========
+proto-route uses [PVP Versioning][1].
+The change log is available [on GitHub][2].
```

Comment:
[...][1] or [...][2] should point to some links. But links are missing in this file.